openmetadata-ui/src/main/resources/ui/src/utils/Fqn.ts

```diff
@@ -66,4 +66,11 @@ export default class Fqn {
 
     throw new Error(`${i18next.t('label.invalid-name')} ${name}`);
   }
+
+  // Get the name without quotes
+  // Useful for getting the service name from the fqn in the url
+  // Example: "service.Name/example" -> service.Name/example
+  static getNameWithoutQuotes(fqn: string) {
+    return fqn.replace(/^"|"$/g, '');
+  }
```

Comment:
Not sure what's user case for having this

Reply:
What's issue in preserving quotes?

Reply:
I'll use the service name here instead. Just to be safe